schemas/queries/prometheus/query.cue

```diff
@@ -21,8 +21,9 @@ spec: {
 	plugin: {
 		kind: "PrometheusTimeSeriesQuery"
 		spec: close({
-			datasource: {
+			datasource?: {
 				kind: prometheus.kind
+				name?: string
```

Comment:
this doesnt add anything new compared to base def

Reply:
ah yeah you are right. I forgot about the bas def. My bad